overview/typescript.md

```diff
@@ -0,0 +1,38 @@
+---
+title: TypeScript
+description: TypeScript is an open-source language which builds on JavaScript
+sidebar_position: 8
+---
+
+## Overview
+
+TypeScript is an open-source language which builds on JavaScript, one of the world’s most used tools, by adding static type definitions.
+
+Types provide a way to describe the shape of an object, providing better documentation, and allowing TypeScript to validate that your code is working correctly.
+
+## TypeScript and Angular
+
+Angular applications can only be built with TypeScript.
+As Angular is the predominant language used for PlaceOS Frontend Applications this makes TypeScript a requirement.
```

Comment:
Frontend should be lower case if talking about frontend applications in general, or pluralised if it is talking about 
https://github.com/PlaceOS/frontends

Reply:
I have changed this to lower case but not plural, these docs are designed for anyone to write their own front end and may not be using and/or referencing https://github.com/PlaceOS/frontends

Reply:
perfect